build_tools/packaging/linux/package.json

```
@@ -0,0 +1,184 @@
+
+[
+ {
+   "Package" : "rocm-core",
+   "Version" : "7.1.0",
+   "Architecture" : "amd64",
+   "BuildArch" : "x86_64",
+   "DEBDepends" : [ "python3", "libc6"],
+   "RPMRequires" : [ "python3", "glibc"],
+   "Maintainer" : "ROCm Dev Support <rocm-dev.support@amd.com>",
+   "Description" : "ROCM-CORE package helps to get ROCm release version and install path",
+   "Section" : "devel",
+   "Priority" : "optional",
+   "Group" : "unknown",
+   "License" : "MIT",
+   "Vendor" : "Advanced Micro Devices, Inc",
+   "Homepage" : "https://github.com/AMD-ROCm-Internal/rocm-core",
+   "Components" :["lib", "dev", "doc" ],
+   "Artifact" : "base",
+   "Gfxarch" : "false"
+ },
+ {
+   "Package" : "rocm-cmake",
+   "Version" : "7.1.0",
+   "Architecture" : "amd64",
+   "BuildArch" : "x86_64",
+   "DEBDepends" : [],
+   "RPMRequires" : [],
+   "Maintainer" : "ROCm Dev Support <rocm-dev.support@amd.com>",
+   "Description" : "rocm-cmake built using CMake",
+   "Section" : "devel",
+   "Priority" : "optional",
+   "Group" : "unknown",
+   "License" : "MIT",
+   "Vendor" : "Advanced Micro Devices, Inc",
+   "Homepage" : "https://github.com/ROCm/rocm-cmake",
+   "Components" :["lib", "dev", "doc" ],
+   "Artifact" : "base",
+   "Gfxarch" : "false"
+ },
+ {
+   "Package" : "rocm-smi-lib",
+   "Version" : "7.1.0",
+   "Architecture" : "amd64",
+   "BuildArch" : "x86_64",
+   "DEBDepends" : [ "python3", "libc6"],
+   "RPMRequires" : [ "python3", "glibc"],
+   "Maintainer" : "RocmSMILib Support <rocm-smi.support@amd.com>",
+   "Description" : "AMD System Management libraries",
+   "Section" : "devel",
+   "Priority" : "optional",
+   "Group" : "unknown",
+   "License" : "MIT",
+   "Vendor" : "Advanced Micro Devices, Inc",
+   "Homepage" : "https://github.com/AMD-ROCm-Internal/rocm_smi_lib",
```

public URL: https://github.com/ROCm/rocm_smi_lib, recently moved to https://github.com/ROCm/rocm-systems

```suggestion
   "Homepage" : "https://github.com/ROCm/rocm-systems/tree/develop/projects/rocm-smi-lib",
```


```suggestion
   "Homepage" : "https://github.com/ROCm/rocm-systems/",
```